extensions/wikia/Rail/scripts/Rail.js

```diff
@@ -37,6 +37,18 @@ $(function () {
 			params.excludeScss = window.wgSassLoadedScss;
 		}
 
+		// TODO XW-2760 remove after experiment is done
+		if (
+			window.Wikia &&
+			window.Wikia.AbTest &&
+			true
+			// FIXME uncomment and adjust after AB test is set up
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1134)